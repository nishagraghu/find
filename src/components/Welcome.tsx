import React from 'react';
import { View,  StyleSheet } from 'react-native';
import { Text } from 'react-native-paper';
import { welcomeText, getStartedText, startButtonText } from '../utils/constants';
const Welcome: React.FC = () => {
    return (
        <View style={styles.welcome}>
            <Text variant="titleLarge" style={styles.text}>{welcomeText}</Text>
            <Text variant="titleLarge"  style={styles.text}>{getStartedText}</Text>
            <Text variant="titleLarge" style={styles.text}>{startButtonText}</Text>
        </View>
    );
};

const styles = StyleSheet.create({
    container: {
        flex: 1,

        backgroundColor: '#fcfcfc',
        alignItems: 'center',
        justifyContent: 'center',
    },
    container1: {

        flex: 0.6,

    },
    container2: {
        flex: 0.4,

    },
    text: {

        textAlign: 'center',
        textAlignVertical: 'center',
    },
    welcome: {
        width:500,
        textAlign: 'center',
        // fontWeight: 24,
        marginLeft: 30,
        textAlignVertical: 'center',

    }
});

export default Welcome;