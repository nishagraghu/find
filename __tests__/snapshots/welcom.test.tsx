import React from 'react';
import { render } from '@testing-library/react-native';

import Welcome from '../../src/components/Welcome'


describe('Welcome component', () => {
  test('renders correctly', () => {
    const { toJSON } = render(<Welcome />);
    expect(toJSON()).toMatchSnapshot();
  });
});
