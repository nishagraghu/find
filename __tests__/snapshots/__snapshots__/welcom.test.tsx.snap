// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome component renders correctly 1`] = `
<View
  style={
    {
      "marginLeft": 30,
      "textAlign": "center",
      "textAlignVertical": "center",
      "width": 500,
    }
  }
>
  <Text
    style={
      [
        {
          "color": "rgba(28, 27, 31, 1)",
          "fontFamily": "System",
          "fontSize": 22,
          "fontWeight": "400",
          "letterSpacing": 0,
          "lineHeight": 28,
        },
        {
          "textAlign": "left",
        },
        {
          "writingDirection": "ltr",
        },
        {
          "textAlign": "center",
          "textAlignVertical": "center",
        },
      ]
    }
  >
    Welcome to GETSPARES!
  </Text>
  <Text
    style={
      [
        {
          "color": "rgba(28, 27, 31, 1)",
          "fontFamily": "System",
          "fontSize": 22,
          "fontWeight": "400",
          "letterSpacing": 0,
          "lineHeight": 28,
        },
        {
          "textAlign": "left",
        },
        {
          "writingDirection": "ltr",
        },
        {
          "textAlign": "center",
          "textAlignVertical": "center",
        },
      ]
    }
  >
    To get started,
  </Text>
  <Text
    style={
      [
        {
          "color": "rgba(28, 27, 31, 1)",
          "fontFamily": "System",
          "fontSize": 22,
          "fontWeight": "400",
          "letterSpacing": 0,
          "lineHeight": 28,
        },
        {
          "textAlign": "left",
        },
        {
          "writingDirection": "ltr",
        },
        {
          "textAlign": "center",
          "textAlignVertical": "center",
        },
      ]
    }
  >
    Press the Start button below
  </Text>
</View>
`;
